prefab/prefab-publishing/README.md

```diff
@@ -1,27 +1,55 @@
-# Prefab AAR Publishing
+prefab-publishing
+=================
 
-The prefab-publishing sample shows how to export native libraries in an AAR
-using the [Prefab] format. This sample build a trivial library `mylibrary` that
-exports the `void my_api();` function to be used by projects that import this
-AAR.
-
-To export native libraries in an AAR with Android Gradle Plugin 4.1+, make the
-following changes to your Android Library project's [build.gradle]:
+This sample shows how to export native libraries to an AAR
+using the [Prefab] format. Generally speaking, to export native libraries in an AAR 
+with Android Gradle Plugin 4.1+, make the following changes to your Android Library project's [build.gradle]:
 
 * Enable the `prefabPublishing` build feature
+```
+    android.buildFeatures.prefabPublising true
+```
+
 * Declare the libraries and headers you wish to export in the `android.prefab`
   block
+```
+    android.prefab.mylibrary {
+        headers "src/main/cpp/include"
+    }
+```
+* (Optional) If the exported native modules are **not** used by the Kotlin/Java modules in this AAR,
+  you could avoid [the unnecessary duplication inside AAR]
+```
+    packagingOptions {
+        exclude("**/libmylibrary.so")
+        exclude("**/libc++_shared.so")
+    }
+```
+This sample builds a trivial library called `mylibrary` (libmylibrary.so), and packs it into 
+mylibrary.aar. From here you could distribute the AAR via Maven Center or in other ways.
+For how to use the generated dependency AAR, refer to [the prefab-dependency sample] in this repo;
+for the official documentation about Prefab in AGP, go to [C/C++ dependencies] website.
```

Comment:
see the [C/C++ dependencies] documentation.

(it's a web page, not a web site)